packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -47,8 +49,11 @@ exports.sourceNodes = async (
   },
   pluginOptions
 ) => {
-  const { createNode, deleteNode, touchNode, setPluginStatus } = actions
-
+  const { createNode, deleteNode, touchNode } = actions
+  const client = createClient({
+    space: `none`,
```

Comment:
@axe312ger is this intentional ? should we pull the token and the space id from the plugin option

Reply:
@Khaledgarbaya I just used what was in the PR, probably your own code :P


I think we need to pass them otherwise `createClient` would fail to to missing config options. But as we do not connect to the space and use the SDK just for resolving the "fake" response, we can pass any space id and token

Reply:
🤦 make total sense